account_tax_analysis/account_tax_analysis_view.xml

```diff
@@ -26,18 +26,27 @@
           <newline/>
           <group expand="1" string="Group By...">
             <filter string="Partner" icon="terp-partner"
+            		name="group_by_partner"
                     domain="[]"
                     context="{'group_by': 'partner_id'}"/>
             <separator orientation="vertical"/>
             <filter string="Account"
+            		name="group_by_account"
                     icon="terp-folder-green"
                     context="{'group_by': 'account_id'}"/>
             <separator orientation="vertical"/>
             <filter string="Period"
+            	    name="group_by_period"
                     icon="terp-go-month"
                     domain="[]"
                     context="{'group_by': 'period_id'}"/>
+            <filter string="Account / Tax"
```

Comment:
This should be put as a favorite instead of a predefined group, because it's a combination of two groups